Assignment1_q2_Node.py

```diff
@@ -0,0 +1,16 @@
+# Node is a unit which is used in my implementation of Singly Linked List
+class Node(object):
+
+    def __init__(self, data = None, next_node = None):
+        # Node is storing some data and a pointer to the next node
+        self.data = data
```

Comment:
I haven't seen the syntax of private variables in python before, thanks for this suggestion! I have changed it now